ci-operator/config/openshift/oc/openshift-oc-master.yaml

```diff
@@ -19,10 +19,9 @@ images:
       as:
         - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
   to: cli
-# Do not promote until migrated from staging
-#promotion:
-#  name: "4.2"
-#  namespace: ocp
+promotion:
```

Comment:
wrong file?  SDN pull contains oc change?

Reply:
thx, goland saved it from my previous PR